src/main/java/com/alipay/oceanbase/rpc/protocol/payload/impl/execute/aggregation/ObTableAggregation.java

```diff
@@ -0,0 +1,145 @@
+        this.tablequery.addAggregation(ObTableAggregationType.MIN, columnName);
+        message.put(getIndex(), "min(" + columnName + ")");
+        updateIndex();
+    }
+
+    public void min(String columnName, String aliasName) {
+        this.tablequery.addAggregation(ObTableAggregationType.MIN, columnName);
+        message.put(getIndex(), "min(" + aliasName + ")");
+        updateIndex();
+    }
+
+    public void max(String columnName) {
+        this.tablequery.addAggregation(ObTableAggregationType.MAX, columnName);
+        message.put(getIndex(), "max(" + columnName + ")");
+        updateIndex();
+    }
+
+    public void max(String columnName, String aliasName) {
+        this.tablequery.addAggregation(ObTableAggregationType.MAX, columnName);
+        message.put(getIndex(), "max(" + aliasName + ")");
+        updateIndex();
+    }
+
+    public void count() {
+        this.tablequery.addAggregation(ObTableAggregationType.COUNT, "*");
+        message.put(getIndex(), "count(*)");
+        updateIndex();
+    }
+    public void count(String aliasName) {
+        this.tablequery.addAggregation(ObTableAggregationType.COUNT, "*");
+        message.put(getIndex(), "count(" + aliasName + ")");
+        updateIndex();
+    }
+
+    public void sum(String columnName) {
+        this.tablequery.addAggregation(ObTableAggregationType.SUM, columnName);
+        message.put(getIndex(), "sum(" + columnName + ")");
+        updateIndex();
+    }
+
+    public void Sum(String columnName, String aliasName) {
+        this.tablequery.addAggregation(ObTableAggregationType.SUM, columnName);
+        message.put(getIndex(), "sum(" + aliasName + ")");
+        updateIndex();
+    }
+
+    public void avg(String columnName) {
+        this.tablequery.addAggregation(ObTableAggregationType.AVG, columnName);
+        message.put(getIndex(), "avg(" + columnName + ")");
+        updateIndex();
+    }
+
+    public void avg(String columnName, String aliasName) {
+        this.tablequery.addAggregation(ObTableAggregationType.AVG, columnName);
+        message.put(getIndex(), "avg(" + aliasName + ")");
+        updateIndex();
+    }
+
+    public void setScanRangeColumns(String... columns) {
+        this.tablequery.setScanRangeColumns(columns);
+    }
+
+    public void addScanRange(Object start, Object end) {
+        this.tablequery.addScanRange(start, end);
+    }
+
+    public void addScanRange(Object[] start, Object[] end) {
+        this.tablequery.addScanRange(start, end);
+    }
+
+    public void indexName(String indexName) {
+        this.tablequery.indexName(indexName);
+    }
+
+    public void setFilter(ObTableFilter filter) {
+        this.tablequery.setFilter(filter);
+    }
+
+    public void setOperationTimeout(long operationTimeout) {
+        this.tablequery.setOperationTimeout(operationTimeout);
+    }
+
+    public ObTableAggregationResult execute() throws Exception {
+        if (this.getIndex() == 0) {
```

Comment:
I guess we could access index directly instead of using **getIndex()**

Reply:
fixed